validator/validation/validation.py

```diff
@@ -3,6 +3,7 @@
 from os import path
 from re import sub as regex_sub
 import uuid
+import numpy as np
```

Comment:
This is used in your commented out part, I have added a comment below. Also,  even though it's not from your Pr, `ISMN_Interface` is imported but unused, should it be removed?